### ISSUE TITLE
Installing KDE plasma changed sway cursor

### ISSUE THREAD

**Issue by halikular** [swaywm/sway#6676]
### System information
- **Sway Version**
  - `1.6.1`

- **Debug Log**
   - http://ix.io/3FEa

- **Configuration File**
  - http://ix.io/3FEi

- **Description**
  - I installed the KDE [plasma-meta](https://archlinux.org/packages/extra/any/plasma-meta/) package and [plasma-wayland-session](https://archlinux.org/packages/extra/x86_64/plasma-wayland-session/) for wayland support with the AUR helper paru by running:
  - `$ paru -S plasma-meta plasma-wayland-session`
  - After enabling sddm.service and rebooting into KDE plasma. Then disabling sddm.service and rebooting back into sway the cursor had changed to the KDE plasma theme on graphical applications like Firefox and doom Emacs Wayland or XWayland. However when the cursor is on the desktop (wallpaper) it returns to the default sway cursor, placing the cursor over alacritty also uses the default cursor theme. I tried to explicitly set the default cursor in sway config:
  - `seat seat0 xcursor_theme default`
  - It didn't seem to do anything after exiting and starting sway again.

**Comment by emersion** [comment]
It probably changed the GTK theme. Closing because Sway can't do anything about it. Use a tool to change the GTK theme back if you need.

**Comment by halikular** [comment]
Yes I realized afterwards this isn't a bug of sway. KDE changed the configuration. I deleted `/usr/share/icons/default/index.theme` and `~/.config/gtk-3.0/settings.ini`but now I got the ugly x11 cursor instead of the sway/wayland cursor. This is a relevant issue here since the same thing might happen to others when installing additional desktop environments. I can't find any documentation on this "sway" cursor, and it might not even be something provided by sway, since I've only had it on Arch Linux but on NixOS it was the x11 cursor.
